.env.example

```diff
@@ -20,6 +20,9 @@ VUE_APP_AUTH_API_URL="https://auth-api-dev-142173140222.northamerica-northeast1.
 VUE_APP_AUTH_API_VERSION="/api/v1"
 VUE_APP_LEGAL_API_URL="https://legal-api-dev.apps.silver.devops.gov.bc.ca"
 VUE_APP_LEGAL_API_VERSION_2="/api/v2"
+VUE_APP_BUSINESS_API_GW_URL="https://test.api.connect.gov.bc.ca/business-dev"
+VUE_APP_BUSINESS_API_VERSION_2="/api/v2"
+VUE_APP_BUSINESS_API_KEY=""
```

Comment:
These need to be added to your .env file as well (and the key populated). Do not commit any files containing the API key!